### ISSUE TITLE
does not work with poetry 1.0.0b1

### ISSUE THREAD

**Issue by jasonkuhrt** [UnitedIncome/serverless-python-requirements#393]
Apps have started to fail at deploy with error like:

```
  ENOENT: no such file or directory, link '/Users/jasonkuhrt/projects/dialoguemd/beautiful-mind/mattermost-posts-consumer/requirements.txt' -> '/Users/jasonkuhrt/projects/dialoguemd/beautiful-mind/mattermost-posts-consumer/.serverless/requirements.txt'
```

Reason we found was that poetry had been upgraded to latest preview version, [`1.0.0b1`](https://github.com/sdispater/poetry/releases). Rolling poetry back to `1.0.0a5` resolved the issue. Creating issue here but it may be that it belongs on `poetry` side.

**Comment by mokamoto12** [comment]
The default behavior of the `poetry export` has changed.
If the `-o` option is not specified, the contents are printed to standard output.

This problem will be fixed in #395

**Comment by bsamuel-ui** [comment]
Closed as #395 is merged. Happy to reopen if there are still issues.
